src/commands/docs.js

```diff
@@ -27,10 +27,57 @@ const fs = require('fs');
 const path = require('path');
 
 /**
- * Command to create docs from .EO sources.
+ * Recursively reads all .xmir files from a directory.
+ * @param {string} dir - Directory path
+ * @return {string[]} Array of file paths
+ */
+function readXmirFilesRecursively(dir) {
+  const files = [];
+  const entries = fs.readdirSync(dir, {withFileTypes: true});
+
+  for (const entry of entries) {
+    const fullPath = path.join(dir, entry.name);
+    if (entry.isDirectory()) {
+      files.push(...readXmirFilesRecursively(fullPath));
+    } else if (entry.name.endsWith('.xmir')) {
+      files.push(fullPath);
+    }
+  }
+
+  return files;
+}
+
+/**
+ * Command to generate documentation.
  * @param {Hash} opts - All options
  */
 module.exports = function(opts) {
-  const filePath = path.resolve('eodocs.html');
-  fs.writeFileSync(filePath, '');
+  try {
+    const inputDir = path.resolve(opts.target, '.eoc', '1-parse');
+    const outputDir = path.resolve(opts.target, 'docs');
+
+    fs.mkdirSync(outputDir, {recursive: true});
+
+    const xmirFiles = readXmirFilesRecursively(inputDir);
+
+    for (const xmirFile of xmirFiles) {
+      const relativePath = path.relative(inputDir, xmirFile);
+      const packagePath = path.dirname(relativePath).split(path.sep).join('.');
+      const outputPath = path.join(outputDir, `package_${packagePath}.html`);
```

Comment:
@SenjeyB let's try to use variables that consist of one word. It's obvious what's path and what's file. https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html